src/views/RecIM/components/Helpers/index.js

```diff
@@ -1,35 +1,28 @@
-import { DateTime } from 'luxon';
+import { isValid, parseISO, format } from 'date-fns';
 
 const standardDate = (date, includeTime) => {
-  if (!DateTime.isDateTime(date)) date = DateTime.fromISO(date);
-  let formattedDate = date.monthShort + ' ' + date.day;
-  if (includeTime) {
-    formattedDate += ' ' + date.toLocaleString(DateTime.TIME_SIMPLE);
-  }
+  if (!isValid(date)) date = parseISO(date);
```

Comment:
Native JS Date already supports parsing ISO (in fact it's the only officially supported format for parsing dates from strings)
```suggestion
  if (!isValid(date)) date = Date.parse(date);
```

Reply:
You probably don't need to use `parseISO` anywhere, so it would be great if you could update the other uses of it as well.